test/functional/full_text_search/search_controller_test.rb

```diff
@@ -215,13 +215,18 @@ def format_api_results(items, total_count: nil)
           datetime ||= changeset.committed_on
         end
         datetime ||= item.updated_on if item.respond_to?(:updated_on)
-        datetime ||= item.created_on if item.respond_to?(:created_on)
+        if item.respond_to?(:created_on)
+          datetime ||= item.created_on
```

Comment:
Could you rename the local varialble name to `last_modified_at` from `datetime`?

Reply:
fix: [3bd7a5e](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/3bd7a5eb1255c510aca5e45f4c0cca26921e069c)